src/zgossip.c

```diff
@@ -520,6 +555,65 @@ zgossip_test (bool verbose)
     zactor_destroy (&alpha);
     zactor_destroy (&beta);
 
+#ifdef CZMQ_BUILD_DRAFT_API
+    // curve
+    if (zsys_has_curve()) {
+        printf("testing CURVE support");
+        zclock_sleep (2000);
```

Comment:
why the 2 seconds sleep before the tests? did you have any issues?

Reply:
most of this dev took place on vbox (vagrant, ubuntu, etc). i've noticed a few issues with doing this on a vm over the years where some race like issues crop up and timers seem to make things happy again. i'll retry (now that i know it works end to end locally) and see if it's still needed. sometimes the crypto needed time to settle too [again, on the vm in  some odd cases].

Reply:
Yeah I've seen a few as well, if you check some tests like zproxy do extra sleeps in the CURVE case too